recipes/opuslib/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "opuslib" %}
+{% set version = "3.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2cb045e5b03e7fc50dfefe431e3404dddddbd8f5961c10c51e32dfb69a044c97
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - libopus
+  host:
+    - pip
+    - python
+    - libopus 
+  run:
+    - python
+    - libopus
```

No need for a build section, only need `libopus` for run, and need to add a Python version constraint for noarch python:
```suggestion
  host:
    - pip
    - python >=3.6
  run:
    - python >=3.6
    - libopus
```